app/src/app.ts

```
@@ -36,25 +36,67 @@ const vueApp = createApp({ render: () => h(App) })
+    console.error('Failed to initialize Flux:', error);
   }
 })();
 
-// Mount the app
-vueApp.mount('#app');
-
-// Check for service worker updates every 10 minutes and reload
+// Service worker registration (call composable unconditionally, but only register in standalone mode)
 const intervalMS = 60 * 10 * 1000;
-const updateServiceWorker = useRegisterSW({
+const { updateServiceWorker } = useRegisterSW({
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)